index.js

```diff
@@ -1,22 +1,18 @@
 var pull = require('pull-stream');
 var toPull = require('stream-to-pull-stream');
 var datDefaults = require('dat-swarm-defaults');
-var swarm = require('discovery-swarm');
+var swarm = require('hyperswarm');
+var crypto = require('crypto');
 
 function createPeer(_opts) {
   var swarmOpts = Object.assign({}, _opts || {});
   delete swarmOpts.key;
   delete swarmOpts.keys;
-  swarmOpts.dns = typeof swarmOpts.dns === 'undefined' ? false : swarmOpts.dns;
 
-  var port = swarmOpts.port || 0;
-  delete swarmOpts.port;
+  //swarmOpts.bootstrap = ['ws://hyperswarm.mauve.moe']
+  //swarmOpts.bootstrap = ['ws://localhost:4977']
```

Comment:
Are these going to be uncommented? Is it required that we provide at least some bootstrap server? (If so, the README needs to be updated, and preferably, throw an error if it's missing)

Reply:
I don't think they need to be.  Ideally the user of multiserver-dht *should* provide them, but both Hyperswarm and Hyperswarm-Web provide default servers which are used if nothing else is provided.  Those could probably just be removed, or left in as suggestions.